.circleci/config.yml

```diff
@@ -93,7 +92,7 @@ jobs:
 
             source pyodide-env/bin/activate
             export PATH=$PWD/firefox:$PATH
-            pytest test -v --instafail -k firefox
```

Comment:
Why was this removed?  This is really useful if a test times out on Circle-CI -- if we don't do this, we don't get any output from the test (or any other failed tests).

Reply:
Never mind.  I see you've added to the config below.